spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/security/oauth2/resource/reactive/ReactiveOAuth2ResourceServerAutoConfigurationTests.java

```diff
@@ -740,4 +784,15 @@ SecurityWebFilterChain testSpringSecurityFilterChain(ServerHttpSecurity http) {
 
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@EnableWebFluxSecurity
```

Comment:
I don't think `@EnableWebFluxSecurity` is needed here.

Reply:
Removed